docs/src/pages/system/basics/Demo.tsx

```diff
@@ -1,43 +1,144 @@
+        height: ['100px', '200px', '100%', '100%', '100%'],
+      }}
+    >
+      <Box
+        {...props}
+        component="img"
+        sx={{
+          border: 2,
+          borderColor: 'white',
+          display: 'block',
+          position: 'relative',
+          height: imgSize,
+          width: imgSize,
+          margin: 'auto',
+          borderRadius: '50%',
+          ...props.sx,
+        }}
+      />
+    </Box>
+  );
+};
+
+interface CardContentProps {
+  color?: Color;
+  header?: string;
+  description?: string;
+  sx?: SxProps;
+}
+
+const CardContent: React.FC<CardContentProps> = (props) => {
+  return (
+    <Box
+      {...props}
+      sx={{
+        display: 'inline-block',
+        px: 1,
+        py: 2,
+        ...props.sx,
+      }}
+    >
+      <Box
+        className="CardContent-header"
+        sx={{
+          fontSize: [12, 14, 16, 18, 20],
+          fontWeight: [400, 400, 400, 500, 600],
+          color: `${props.color}.main`,
+          my: [1, 1, 2, 2, 2],
+        }}
+      >
+        {props.header}
+      </Box>
+      <Box
+        sx={{
+          fontSize: [10, 12, 14, 16, 18],
+          fontWeight: [400, 400, 400, 500, 600],
+          color: 'grey',
+        }}
+      >
+        {props.description}
+      </Box>
+    </Box>
+  );
+};
+
+interface CardProps {
+  color?: 'primary' | 'secondary' | 'success' | 'warning' | 'error' | 'info';
+  header?: string;
+  description: string;
+  sx?: SxProps;
+  profileImage?: string;
+}
+
+const Card: React.FC<CardProps> = (props) => {
+  const {
+    color = 'primary',
+    profileImage,
+    header,
+    description,
+    sx,
+    ...rest
+  } = props;
+  return (
+    <Box
+      {...rest}
+      sx={{
+        display: 'flex',
+        flexDirection: ['column', 'column', 'row', 'row', 'row'],
+        width: [100, 200, 300, 400, 500],
+        border: 1,
+        borderColor: 'lightgrey',
+        borderRadius: '5px',
+        boxShadow: 3,
+        minHeight: [150, 100, 200, 300, 300],
+        ':hover': {
+          '& .CardHeader': {
+            bgcolor: `${props.color}.dark`,
+          },
+          '& .CardContent-header': {
+            color: `${props.color}.dark`,
+          },
+          boxShadow: 6,
+        },
+        ...sx,
+      }}
+    >
+      <CardHeader color={color} src={profileImage} alt={header} />
+      <CardContent color={color} header={header} description={description} />
+    </Box>
+  );
+};
 
 export default function Demo() {
   return (
-    <NoSsr>
-      <ThemeProvider theme={theme}>
-        <Box
-          color="primary.main"
-          bgcolor="background.paper"
-          fontFamily="h6.fontFamily"
-          fontSize={{
-            xs: 'h6.fontSize',
-            sm: 'h4.fontSize',
-            md: 'h3.fontSize',
-          }}
-          p={{ xs: 2, sm: 3, md: 4 }}
-        >
-          @material-ui/system
-        </Box>
-      </ThemeProvider>
-    </NoSsr>
+    <Card
```

Comment:
For the demo, I think that we can simplify it, I don't think that it should cover all the features that exists.

- What do you think about this model?

<img width="318" alt="Capture d’écran 2020-11-05 à 20 40 40" src="https://user-images.githubusercontent.com/3165635/98288643-705bdc80-1fa7-11eb-8ca5-4f756785eea9.png">

Found it on https://dribbble.com/shots/14329877-Seller-product-vision.

- How about we only handle two breakpoints: xs and sm or md?
- What about we don't create any intermediary components?

Reply:
I wasn't sure what we want to show in that demo, was trying to make it as compelling as possible :D. Let me create a small PR towards this one with this Demo and compare and we can decide 👍

Reply:
I would argue that we can make it compelling with its simplicity. How can I quickly implement a great design?

Reply:
Here it is https://github.com/mnajdova/material-ui/pull/13 Let me know if it looks and and will merge it.

Reply:
This is already much better. I believe https://github.com/mnajdova/material-ui/pull/14 kills it 😁.